dist/selleckt-legacy.js

```diff
@@ -117,7 +117,7 @@ MultiSelleckt.prototype._mutationHandler = function (mutations){
 
     this.items = this.items.concat(newItems);
 
-    if(removedItems.length){
+    if (removedItems.length) {
```

Comment:
Can you please follow the code style in this repo to make it consistent and the diff smaller 😉  That would be awesome. TA